conjure-java-core/src/test/resources/test/api/TestServiceBlocking.java.dialogue

```diff
@@ -30,46 +30,80 @@ import javax.annotation.Nonnull;
 public interface TestServiceBlocking {
     /**
      * Returns a mapping from file system id to backing file system configuration.
+     * @apiNote GET /catalog/fileSystems
      */
     @Nonnull
     Map<String, BackingFileSystem> getFileSystems(AuthHeader authHeader);
 
     /**
      * foo $bar
+     * @apiNote POST /catalog/datasets
      */
     Dataset createDataset(AuthHeader authHeader, String testHeaderArg, CreateDatasetRequest request);
 
+    /**
+     * @apiNote GET /catalog/datasets/{datasetRid}
+     */
     Optional<Dataset> getDataset(AuthHeader authHeader, ResourceIdentifier datasetRid);
 
+    /**
+     * @apiNote GET /catalog/datasets/{datasetRid}/raw
+     */
     InputStream getRawData(AuthHeader authHeader, ResourceIdentifier datasetRid);
 
+    /**
+     * @apiNote GET /catalog/datasets/{datasetRid}/raw-aliased
+     */
     InputStream getAliasedRawData(AuthHeader authHeader, ResourceIdentifier datasetRid);
 
+    /**
+     * @apiNote GET /catalog/datasets/{datasetRid}/raw-maybe
+     */
     Optional<InputStream> maybeGetRawData(AuthHeader authHeader, ResourceIdentifier datasetRid);
 
+    /**
+     * @apiNote GET /catalog/datasets/{datasetRid}/string-aliased
```

Comment:
I don't think this is valid javadoc since the curly braces aren't escaped.
```suggestion
     * @apiNote {@code GET /catalog/datasets/{datasetRid}/string-aliased}
```

Reply:
From what I can tell it's fine and boils down to whether we're using an `@` sign, i.e. `{thisisfine}` v `{@thisiswrong}`, e.g.:

![image](https://user-images.githubusercontent.com/471097/85888994-8e882380-b7e2-11ea-87ad-33c3d3af8e55.png)

Generates the java below:

![image](https://user-images.githubusercontent.com/471097/85889029-9a73e580-b7e2-11ea-89d4-91dad46652de.png)

(when using ` -tag "apiNote:a:API Note:"`)

Reply:
:+1:

Reply:
(actually `@code` gives us fixed-width fonts in the javadoc, so I'll update :))

Reply:
Thanks :-)